lib/package_json.ts

```diff
@@ -94,7 +94,7 @@ export function getPackageJson({
       // override with specified scripts
       ...(packageJsonObj.scripts ?? {}),
     })
-    : packageJsonObj.scripts;
+    : packageJsonObj.scripts ?? {};
```

Comment:
theoretically we could remove this one here, as it gets removed further down by `deleteEmptyKeys`, however the other branch of the condition has the fallback and we want this to also be there if the code below changes at a later point in time.